app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
11. Deleting all files that are never used. If file that no have somethings in it, please delete them, don't push to GIT